{
    "type": "bundle",
    "id": "bundle--7e2c323d-2021-4796-92d6-edf6cb9e51f4",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--2a05c1d4-e36a-483c-a90a-a026c894b097",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2024-09-12T01:06:43.471058Z",
            "modified": "2024-09-12T01:06:43.471058Z",
            "name": "CVE-2024-45027",
            "description": "In the Linux kernel, the following vulnerability has been resolved:\n\nusb: xhci: Check for xhci->interrupters being allocated in xhci_mem_clearup()\n\nIf xhci_mem_init() fails, it calls into xhci_mem_cleanup() to mop\nup the damage. If it fails early enough, before xhci->interrupters\nis allocated but after xhci->max_interrupters has been set, which\nhappens in most (all?) cases, things get uglier, as xhci_mem_cleanup()\nunconditionally derefences xhci->interrupters. With prejudice.\n\nGate the interrupt freeing loop with a check on xhci->interrupters\nbeing non-NULL.\n\nFound while debugging a DMA allocation issue that led the XHCI driver\non this exact path.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2024-45027"
                }
            ]
        }
    ]
}